mycroft/messagebus/message.py

```diff
@@ -49,11 +50,11 @@ def serialize(self):
         Returns:
             str: a json string representation of the message.
         """
-        return json.dumps({
-            'type': self.type,
-            'data': self.data,
-            'context': self.context
-        })
+        return json.dumps(dict(
+            type=self.type,
+            data=self.data,
+            context=self.context
+        ))
```

Comment:
Question about this is there an advantage to using the dict() rather than {}? Or is it just a style thing?

Reply:
It is a style thing.  IMO, when you are declaring a dictionary with string keys (such as in this example), the dict constructor is easier to read (and type).

Reply:
Understood, don't quite agree but don't really mind as long as you're fine with me not doing this :)